src/main/java/business/LoginPageBo.java

```diff
@@ -1,26 +0,0 @@
-package business;
```

Comment:
Don't change duplicate files in this pull request

Reply:
I have reverted dublicates